### PR TITLE
add GCP org hierarchy viewer (gcpohv)

### PR DESCRIPTION
**Pull request by angstwad**
Adds `gohv`, a CLI utility for visualizing your organization hierarchy in the terminal.  In support of b/135755207.

Replaces the problematic PR #271.

### CODE REVIEW

**Comment by morgante** [comment]
Apologies for the delay, reassigning to @AdrienWalkowiak.

File: tools/gcp-org-hierarchy-viewer/gcpohv_cli.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+from googleapiclient import discovery, errors
+
+_ORG = '🏢'
+_FOLDER = '📁'
+_PROJ = '📦'
+
+
+def setup_credentials(credentials_file):
+    if not credentials_file:
+        return
```
Comment (AdrienWalkowiak) [inline tools/gcp-org-hierarchy-viewer/gcpohv_cli.py]: nit: return None would make it more consistent

File: tools/gcp-org-hierarchy-viewer/gcpohv_cli.py [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+import asciitree
+from google.auth import exceptions
+from google.oauth2 import service_account
+from googleapiclient import discovery, errors
+
+_ORG = '🏢'
+_FOLDER = '📁'
+_PROJ = '📦'
+
+
```
Comment (AdrienWalkowiak) [inline tools/gcp-org-hierarchy-viewer/gcpohv_cli.py]: I know it's a small script, but could you add a small header for each function (except main, maybe)?
Something like: 

function name - high-level description
input parameters, if any
outputs,  if any

This makes it easier to dive into the code, without reading the whole script

File: tools/gcp-org-hierarchy-viewer/gcpohv_cli.py [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    return crm_v1, crm_v2
+
+
+def resource_filter(value, resource, field='displayName'):
+    if field == 'name':
+        return resource[field].split('/')[1] == value
+    return resource[field] == value
+
+
+class Cli:
```
Comment (AdrienWalkowiak) [inline tools/gcp-org-hierarchy-viewer/gcpohv_cli.py]: nit: same here, could you add a high-level doc of the class?

**Pull request merged**